css-ui-4/Overview.bs

```diff
@@ -2135,6 +2135,22 @@ Switching appearance: the 'appearance' property</h3>
 	giving them a <dfn export>primitive appearance</dfn>
 	where CSS can be used to restyle them.
 
+	Using ''appearance: base'' gives <a>widgets</a> a <dfn export>base
+	appearance</dfn>. <a>Widgets</a> in <span>base appearance</span> have their
+	<a>native appearance</a> suppressed where CSS can be used to restyle them,
+	just like <a>primitive appearance</a>, but also have default styles which are
+	usable and interoperable. This is unlike <a>primitive appearance</a> which
+	can cause some <a>widgets</a> to disappear completely.
+
+	A widget can have some style sheet associated with it and those styles can
+	change based on whether the widget has a <span>base appearance</span>.
+	<span>Native appearance</span> and <span>primitive appearance</span> should
+	have the same styles.
+
+	ISSUE: Do not to ship ''appearance: base'' until its stylesheet has been
+	properly and thoroughly defined (and implemented per spec) for all form
+	controls.
```

Comment:
"Do not to ship" needs to be reworded somehow.  But I'd suggest a slightly larger change:

```
  ''appearance:base'' is not ready to ship until its stylesheet has been fully defined
  in the HTML specification, and implemented accordingly, for all HTML form controls.
```

This proposed rewording also makes it clear that (a) this is going to be defined in the HTML spec and (b) the requirement is specific to all HTML form controls (and not any other theoretical form controls).

Reply:
Thanks, I replaced the issue text with your proposed text